bibliopixel/animation/sequence.py

```diff
@@ -1,14 +1,29 @@
 from . import collection
+import random as rand
 
 
 class Sequence(collection.Collection):
+    def __init__(self, layout, random=False, **kwds):
```

Comment:
Now allow randomization. For the new automatic demo mode and just in general. A sequence is boring if it's always in the same order. No it can be random.